src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const moneyPerDay = 40;
```

Comment:
It can also be named with uppercase as other consts
```suggestion
  const COST_PER_DAY = 40;
```